spec/src/main/asciidoc/core/inheritance.asciidoc

```diff
@@ -29,6 +29,8 @@ Then the second bean inherits, and may not override, the qualifiers and bean nam
 The second bean is able to serve the same role in the system as the first.
 In a particular deployment, only one of the two beans may fulfill that role.
 
+Specialization is only present in {cdi_full} and is specified therein.
```

Comment:
Maybe add a link to the chapter?

Reply:
Can do. I think back-links from Full to Lite are very important -- the other way, not so much.